common/src/main/java/com/wynntils/models/guild/GuildModel.java

```diff
@@ -437,6 +437,10 @@ private void parseDiplomacyContent(List<ItemStack> items) {
         WynntilsMod.info("Successfully parsed tributes for guild " + guildName);
     }
 
+    public Set<String> getGuildMembers() {
+        return guildMembers;
```

Comment:
```suggestion
        return Collections.unmodifiableSet(guildMembers);
```

Still keep this even when adding the `isGuildMember` method to the model